development/releasing.md

```diff
@@ -23,16 +23,16 @@ Once all issues have passed acceptance testing and have been merged into `master
 @core_devs I've just created the `<major>.<minor>.x` release branch. Please be aware that any further changes intended for this release will have to be merged to `master` then backported. Thanks!
   ```
 2. Build a beta named `<major>.<minor>.<patch>-beta.1` by pushing a git tag and when CI completes successfully notify the QA team that it's ready for release testing.
-3. [Import translations keys](translations.md#adding-new-keys) into POE and notify the #translations Slack channel translate new and updated values, for example:
+3. Get any new phrases ready for the translations team. Use [the CLI](https://github.com/medic/cht-core/tree/master/scripts/poe) to upload `messages-en.properties`. Then notify the #translations Slack channel translate new and updated values, for example:
   ```
-@channel I've just updated the translations in POE. These keys have been added: "<added-list>", and these keys have been updated: "<updated-list>"
+@channel X.Y.Z has entered release testing, and I have updated the POE editor with new and changed translations, ready to be translated!"
   ```
-4. Create a new document in the [release-notes folder](https://github.com/medic/medic/tree/master/release-notes) in `master`. Ensure all issues are in the GH Project, that they're correct labelled, and have human readable descriptions. Use [this script](https://github.com/medic/medic/blob/master/scripts/changelog-generator) to export the issues into our changelog format. Manually document any known migration steps and known issues. Provide description, screenshots, videos, and anything else to help communicate particularly important changes.
+4. Create a new document in the [release-notes folder](https://github.com/medic/cht-core/tree/master/release-notes) in `master`. Ensure all issues are in the GH Project, that they're correct labelled, and have human readable descriptions (REVIWER: HOW!?!?!). Use [this script](https://github.com/medic/cht-core/blob/master/scripts/changelog-generator) to export the issues into our changelog format. Manually document any known migration steps and known issues. Provide description, screenshots, videos, and anything else to help communicate particularly important changes.
```

Comment:
> (REVIWER: HOW!?!?!)

Specifically, this note refers to guidance as to how to tell that all issues are in the GH project. As this seems complicated / not possible it would be good to either script this (if we've done it in the past by scraping git), document it more clearly or drop the instruction

Reply:
I guess drop the bit about "Ensure all issues are in the GH Project"... we're usually pretty good with that, and as you say it's fairly hard to do.